src/api/loader.coffee

```diff
@@ -58,10 +58,12 @@ handleAPIEvent = (apiEventChannel, baseUrl, setting, requestEvent = {}) ->
     $.ajax(apiSetting)
       .then((responseData) ->
         delete LOADING[apiSetting.url]
-        completedEvent = interpolate(setting.completedEvent, requestEvent.data)
-        completedData = getResponseDataByEnv(isLocal, requestEvent, responseData)
-        apiEventChannel.emit(completedEvent, completedData)
-
+        try
```

Comment:
In principle I agree with you.  However I'm afraid there might be endpoints that do return an empty response by design, so I thought it safer to keep the change as isolated as possible.

I do think it's a good idea to wrap the response handlers in a try/catch so that we can display any errors.  Axiom did this for us by default, but jQuery's promises do not (apparently).